src/components/abstracts.py

```diff
@@ -76,6 +84,7 @@ class AbstractActor(Base):
     V = 0
     w = Base.BASE_BLOCK
     h = Base.BASE_BLOCK
+    weapon_slot: AbstractItem
```

Comment:
This should be an instance attribute not a class attribute
meaning, define it in the `__init__(self)` method
it wouldn't necessarily be the same across all instances putting it in the constructor is a good way to prevent that undesired behavior

https://www.digitalocean.com/community/tutorials/understanding-class-and-instance-variables-in-python-3